src/app/shell/main/filters-list/filters-list.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="filter">
+    <h2 class="filter__header">Знайди найкращі гуртки у своєму місті!</h2>
+    <div class="filter__searchbar">
+        <input class="filter__searchbar-input" type="text"  placeholder="Баскетбол / Динамо / ЦТДЮ / Драма / JavaScript"> 
```

Comment:
I can see that some of you use 2-space tab and some 4-space tab, please be consistent. I would suggest using 2 spaces

Reply:
I've deleted package-lock, looks like I accidentally install it